blocks/bricks/__init__.py

```diff
@@ -444,6 +444,32 @@ class Softmax(Activation):
     def apply(self, input_):
         return tensor.nnet.softmax(input_)
 
+    @application
+    def categorical_cross_entropy(self, x, y):
+        """
+        Return computationally stable softmax cost.
+
+        Parameters
+        ----------
+        :type x: matrix
```

Comment:
Please have a look at the [documentation guidelines](http://blocks.readthedocs.org/en/latest/development/docs.html#docstrings) of Blocks. We use NumPy docstrings, not rST ones. Just look at any of the other bricks for an example.